include/triton/core/tritonbackend.h

```diff
@@ -64,6 +64,7 @@ struct TRITONBACKEND_Response;
 struct TRITONBACKEND_Backend;
 struct TRITONBACKEND_Model;
 struct TRITONBACKEND_ModelInstance;
+struct TRITONBACKEND_BufferAttributes;
```

Comment:
TRITONSERVER_BufferAttributes and the New/Delete/Get/Set methods should all be defined in tritonserver.h

Reply:
Remove this and use TRITONSERVER_BufferAttributes everywhere